data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json

```diff
@@ -159,23 +159,33 @@
   {
     "id": "MISSION_SCIENCE_REP_5",
     "type": "mission_definition",
-    "name": { "str": "Find Lab Tunnels" },
-    "goal": "MGOAL_FIND_ANY_ITEM",
+    "name": "Find Lab Tunnels",
+    "goal": "MGOAL_FIND_ITEM",
     "difficulty": 5,
     "value": 50000,
-    "item": "software_train_data",
-    "start": "reveal_lab_train_depot",
+    "item": "train_data_card",
+    "start": {
+      "assign_mission_target": {
+        "om_terrain": "underground_lab_cargo_stationA",
+        "om_special": [ "Lab_CORE_2x1x3", "lab_mutagen_6_level" ],
+        "reveal_radius": 1,
+        "random": false,
+        "search_range": 200,
+        "z": -4
+      },
+      "effect": [ { "u_spawn_item": "data_card" } ]
+    },
     "origins": [ "ORIGIN_SECONDARY" ],
     "dialogue": {
-      "describe": "We need help…",
```

Comment:
Why are you changing proper ellipsis into three dots? The same for `failure`.

Reply:
That’s not something I meant to do. I’ve fixed that, and I’ll fix the other errors soon.